ckine/figures/figureCommon.py

```diff
@@ -455,9 +454,13 @@ def expScaleWT(predSTAT2, predSTAT15, expSTAT2, expSTAT15, rep2=False):
 
         subExpSTAT2 = np.reshape(subExpSTAT2, (subExpSTAT2.shape[0], subExpSTAT2.shape[1], subExpSTAT2.shape[2], 1))
         subExpSTAT2 = np.tile(subExpSTAT2, (1, 1, 1, subPredSTAT2.shape[3]))
+        subExpSTAT2 = subExpSTAT2[:, 0:2, :, :]
         subExpSTAT15 = np.reshape(subExpSTAT15, (subExpSTAT15.shape[0], subExpSTAT15.shape[1], subExpSTAT15.shape[2], 1))
         subExpSTAT15 = np.tile(subExpSTAT15, (1, 1, 1, subPredSTAT15.shape[3]))
+        subExpSTAT15 = subExpSTAT2[:, 0:2, :, :]
```

Comment:
```suggestion
        subExpSTAT15 = subExpSTAT15[:, 0:2, :, :]
```

Is this a typo?

Reply:
Yep - grabbing that now.

Reply:
It won't change anything as it's just a redundant line

Reply:
Yep - just fixed.